boot.js

```diff
@@ -0,0 +1,145 @@
+const constant = require("./constants");
+const general = require("./schemas/user");
+const owner = require("./schemas");
+require('dotenv').config();
+
+/* Set to true if fake data is needed */
+var populate = true;
```

Comment:
Could we replace var with let? 👻

Reply:
re-arranged the files a bit, changed to let